src/components/Color.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {FC, ReactNode} from 'react';
+import PropTypes from 'prop-types';
+import arrify from 'arrify';
+import chalk from 'chalk';
+
+const methods = [
+	'hex',
+	'hsl',
+	'hsv',
+	'hwb',
+	'rgb',
+	'keyword',
+	'bgHex',
+	'bgHsl',
+	'bgHsv',
+	'bgHwb',
+	'bgRgb',
+	'bgKeyword'
+];
+
+const Color: FC<ColorProps & { children: ReactNode }> = ({children, ...colorProps}) => {
+	if (children === '') {
+		return null;
+	}
+
+	// TODO: Figure out what's going on here and write the type definition
```

Comment:
Is there something better we can do here other than `@ts-ignore` everything?

Reply:
I was going to return to it but forgot. Thank you for bringing it to my attention. I'll look into it in the morning.

Reply:
@twick00 I went back and looked at over the types in Color component. I was able to improve types slightly by consuming types from Chalk instead of having a hardcoded interface for Color props. 

I had to use `as any` in 2 places because no matter what I tried, I couldn't get the correct call signatures to match the prop names. I have a sneaking suspicion that chalk's types are not correct. I looked at trying to fix chalk's types but it's a pretty nuanced API for such a small library. I decided to leave that yak unshaved. I rewrote `transformChildren` to use reduce because I believe it conveys the intention better than `forEach` that was there.

Does this work for you?

Reply:
I restored the original forEach loop because of https://github.com/vadimdemedes/ink/pull/264#discussion_r395977218

Reply:
👍 I'm ok with this.